### PR TITLE
[home] implement projects section for redesigned Expo Go homescreen

### PR DESCRIPTION
**Pull request by FiberJW**
# How

- Added a query to get projects
- Added a section to the feature-flagged home screen to display those projects
- fixed a problem with the development server list item showing an 'enter URL manually' accessory on each item rather than just the end of the list

# Test Plan

I tested this by opening Expo Go on Android and iOS in development mode on my personal account.

### CODE REVIEW

File: apps/bare-expo/ios/Podfile.lock [hunk 1]
```diff
@@ -14,12 +14,9 @@ PODS:
   - EXAmplitude (11.1.0):
     - Amplitude (~> 6.0.0)
     - ExpoModulesCore
-  - EXAppAuth (11.1.0):
```
Comment (ajsmth) [inline apps/bare-expo/ios/Podfile.lock:17]: I'm suspicious of these changes - I would avoid committing this

File: ios/Podfile.lock [hunk 2]
```diff
@@ -1811,14 +1811,11 @@ PODS:
...
     - ExpoModulesCore
-  - EXAppAuth (11.1.0):
-    - AppAuth (~> 1.2)
-    - ExpoModulesCore
   - EXAppleAuthentication (4.1.0):
     - ExpoModulesCore
   - EXApplication (4.0.1):
     - ExpoModulesCore
-  - EXAV (10.2.0):
+  - EXAV (11.0.1):
```
Comment (ajsmth) [inline ios/Podfile.lock:1818]: same here - revert

File: packages/expo-dev-client-components/src/useExpoTheme.ts [hunk 3]
```diff
@@ -11,11 +11,11 @@ export function useCurrentTheme(): 'light' | 'dark' {
 
   let theme: 'light' | 'dark' = 'light';
 
-  if (preference !== 'no-preference') {
+  if (preference && preference !== 'no-preference') {
```
Comment (ajsmth) [inline packages/expo-dev-client-components/src/useExpoTheme.ts:14]: just curious - was preference undefined somewhere?

**Review by ajsmth — APPROVED** [review]
looks good so long as you revert the pod changes

File: packages/expo-dev-client-components/src/useExpoTheme.ts (hunk 3 above)
Comment (FiberJW) [reply packages/expo-dev-client-components/src/useExpoTheme.ts:14]: Yeah, preference sometimes was undefined or an empty string, so I just decided to rely on JS boolean coercion to check if it's there or not.

**Pull request merged**